### PR TITLE
Milestone : Create a data model

### PR DESCRIPTION
**Pull request by Kashif-Saleem-Ghuman**
In this milestone 💪 

- [ ] Created a new app with a meaningful name.
- [ ] Initialized project with Git.
- [ ] Postgres database is set up.
- [ ] -Built project schema.
- [ ] Create and run the necessary migration files.
- [ ] Foreign keys are included.
- [ ] In the posts table the column holding the foreign key to the users table is named author_id
- [ ] All columns that are foreign keys have a corresponding index.

### CODE REVIEW

File: db/migrate/20230329175222_create_post.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class CreatePost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+
+      t.timestamps
+    end
```
Comment (mikethreels) [inline db/migrate/20230329175222_create_post.rb:9]: - [ ] please have a look at the ERD diagram as you can see the Post Table should have a column called likes_counter as well.
afbeelding

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Kashif, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kashif-Saleem-Ghuman** [comment]
@mikethreels  Hi thank you for reviewing and pointing that out. 👍

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Kashif-Saleem-Ghuman 

Your project is complete! Great job for making the changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- migration files ok
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kashif-Saleem-Ghuman** [comment]
@tufoinnkuo10  Thank you for approving my PR 👍

**Pull request merged**